docs/docs/run_minder_server/config_webhook.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Configuring a Webhook
+sidebar_position: 70
+---
+
+# Configuring a Webhook
+Minder requires a webhook to be configured on the repository provider. Currently, Minder only supports GitHub. 
```

Comment:
```suggestion
Minder allows a webhook to be configured on the repository provider to respond to provider events. Currently, Minder only supports GitHub. 
```

We can't say it's required, as we don't set it in the "configuring a provider" section.